tuning/sft_trainer.py

```diff
@@ -129,9 +129,18 @@ def train(
     )
     
     # load the data by parsing JSON
-    json_dataset = datasets.load_dataset('json', data_files=data_args.data_path)
-    formatted_dataset = json_dataset['train'].map(lambda example : {f"{data_args.dataset_text_field}" : example[f"{data_args.dataset_text_field}"] + tokenizer.eos_token})
-    logger.info(f"Dataset length is {len(formatted_dataset)}")
+    data_files = {"train": data_args.data_path}
```

Comment:
Any thoughts on changing the arg to `training_data_path` or something similar? Or should we keep it data_path for compatibility for now?

Reply:
@Ssukriti would appreciate your input on this one, I agree naming to `training_data_path` is better to be descriptive but I know we were wary of changing the name of input parameters that we expose to users since this would affect train conductor as well. We also have a note to update `model_max_length` to be `max_seq_length`
as seen in this [note](https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/tuning/sft_trainer.py#L101-L102)

Reply:
I think we may need to keep it data_path for now . we will make a breaking change once with all the parameter name changes and also update TC image. Thanks for keeping track Anh , once a few PRs are merged after data formatting as well, maybe in end Feb / early March we can have a call to discuss all breaking changes

Reply:
if you like, you could start a box note with the planned changes so we dont lose track

Reply:
created an issue to record the arguments we want to update https://github.ibm.com/ai-foundation/watson-fm-stack-tracker/issues/622